packages/nitro-protocol/contracts/ForceMove.sol

```diff
@@ -312,7 +312,7 @@ contract ForceMove is IForceMove {
         channelStorageHashes[channelId] = _hashChannelData(
             ChannelData(0, uint48(block.timestamp), bytes32(0), address(0), outcomeHash)
         );
-        emit Concluded(channelId);
+        emit Concluded(channelId, uint48(block.timestamp));
```

Comment:
The only other non-null variable we could emit is `outcomeHash`. But that isn't useful unless you have the pre-image.

Reply:
Can you output the preimage? How much gas does that cost?

Reply:
How hard a "no" is that? Is it reasonable to change the function signature to allow this?

Reply:
(the point is to wonder: should our short term strategy be to make it as easy as possible to manage finalized outcomes?)